frontend/src/app/login/page.tsx

```
@@ -0,0 +1,119 @@
+                    placeholder="password"
+                    className="input secondary-theme dark-font w-full"
+                    required
+                  />
+                  <p className="my-3 text-xs">
+                    <Link href="/" className="font-bold underline">
+                      Forgot Password?
+                    </Link>
+                  </p>
+                </div>
+                <div className="mt-5 flex justify-center items-center">
+                  <button className="btn-accent font-medium md:w-40 rounded-lg p-3">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)